src/io/vegaprotocol/DockerisedVega.groovy

```diff
@@ -113,6 +115,9 @@ void run(String command, boolean resume = false) {
     if (resume) {
         extraArguments += ' --resume'
     }
+    if (legacyResume) {
```

Comment:
Should it be legacy? 
![image](https://user-images.githubusercontent.com/1239637/158842698-584b9ba6-46a6-48f4-9886-1af685c3261f.png)